### ISSUE TITLE
Add support for JetStream

### ISSUE THREAD

**Issue by moxit0** [madansp/nats-embedded#4]
Hi, do you plan to add support for JetStream or may be just to add server config method where option like "--js" to be passed ?

**Comment by madansp** [comment]
Added option to enable jetstream. Its available in version 2.1.0 
```                        
 new NatsServerConfig.Builder()
                                .withPort(port)
                                .withEnableJetStream()
                                .withNatsVersion(NatsVersion.LATEST)
                                .build()
```
Store directory can be defined also.
```                       
 new NatsServerConfig.Builder()
                                .withPort(port)
                                .withEnableJetStream("store-dir")
                                .withNatsVersion(NatsVersion.LATEST)
                                .build()
```
Added option to use nats configuration file.
```                         
new NatsServerConfig.Builder()
                                .withPort(port)
                                .withConfigurationFile("nats-config-file.conf")
                                .withNatsVersion(NatsVersion.LATEST)
                                .build()
```

**Comment by moxit0** [comment]
Thank you very much, appreciated it. Now it is possible to test JS related  features.
